assets/asset_manager.cc

```diff
@@ -29,6 +29,10 @@ void AssetManager::PushBack(std::unique_ptr<AssetResolver> resolver) {
   resolvers_.push_back(std::move(resolver));
 }
 
+std::deque<std::unique_ptr<AssetResolver>> AssetManager::TakeResolvers() {
```

Comment:
No idea if this is reasonable :)

Reply:
I'm pretty sure that this is correct because resolvers_ is an lvalue here.

This is behaviorally the same as making a copy of the deque and clearing the original, but the compiler should be able to optimize this to not do the copy now.

@jason-simmons or @chinmaygarde could verify that though.

Reply:
Even with the copy, the most frequent number of elements seems to be 1, and with this change that will grow to 2

Reply:
In case I wasn't clear, this way of doing things _avoids_ a copy.

Reply:
ahh ... good, right :)